pkg/vpcmodel/explainabilityInput.go

```diff
@@ -395,6 +396,8 @@ func (c *VPCConfig) getNodesFromAddress(ipOrCidr string, inputIPBlock *ipblock.I
 	}
 	// internal address
 	networkInterfaces := c.GetNodesWithinInternalAddress(inputIPBlock)
+	// filtering out the nodes which are not represented by their address (currently only LB private IPs):
+	networkInterfaces = slices.DeleteFunc(networkInterfaces, func(n Node) bool { return !n.RepresentedByAddress() })
```

Comment:
Should this specific case be treated differently in the relevant error message?

Reply:
there isnt really a specific case here, it just that GetNodesWithinInternalAddress() might give us some nodes that we do not want to